<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InserModeSelector</class>
 <widget class="QWidget" name="InserModeSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>141</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbSelected">
        <property name="text">
         <string>Selected - for items that are selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbVisible">
        <property name="text">
         <string>Visible - for items that are filtered</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbAll">
        <property name="text">
         <string>All - for all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCurrentRow">
        <property name="text">
         <string>Current row - only current row</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
